gradle/owasp-suppression.xml

```diff
@@ -1,40 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <!-- See https://jeremylong.github.io/DependencyCheck/general/suppression.html for examples -->
-    <suppress until="2024-03-16">
+    <suppress until="2024-06-16">
```

Comment:
Shame that there is still no fix for these

Reply:
yes, I tried up update versions, but it is still reporting it incorrectly.